store/src/test/java/com/zimbra/cs/service/PreviewServletTest.java

```diff
@@ -336,6 +337,20 @@ public void shouldReturnCorrectQueryWhenCalledGenerateQuery() {
     assertEquals("png", query.getOutputFormat().get());
   }
 
+  @Test
+  public void shouldMatchNestedPartNumberFromRequestQuery() {
+    final String requestUrlForPdfPreview = "https://nbm-s01.demo.zextras.io/service/preview/pdf/14473/1.3.2.1.2.3";
```

Comment:
Is `nbm-s01.demo.zextras.io` really necessary or can we put `example.com`?
```suggestion
    final String requestUrlForPdfPreview = "https://example.com/service/preview/pdf/14473/1.3.2.1.2.3";
```